network/src/peer_store/sqlite/peer_store.rs

```diff
@@ -382,6 +385,25 @@ impl PeerStore for SqlitePeerStore {
         self.is_addr_banned(addr)
     }
 
+    fn get_banned_addresses(&self) -> Vec<BannedAddress> {
+        db::get_banned_addresses(&self.conn).expect("get banned addresses")
+    }
+
+    fn insert_ban_address(&mut self, address: IpNetwork, ban_until: u64, ban_reason: &str) {
```

Comment:
`ban address` or `baned_address`? please keep consistency

Reply:
changed to `insert_ban` and `delete_ban`

Reply:
Open for review, naming is hard...